pages/api/user.api.ts

```diff
@@ -0,0 +1,27 @@
+import { APIRequestContext, APIResponse } from '@playwright/test';
+import { UserDTO } from '../../dto/userDTO';
+
+export class UserApi {
+  constructor(
```

Comment:
For the future work(note it for yourself and we'll do it as an additional task)

Try to remove constructor and data from them. 
Also do it in test files: test.beforeEach(async ({ request, baseURL })